app/components/sw-ember-dragula/component.js

```diff
@@ -0,0 +1,5 @@
+import emberDragulaContainer from 'ember-dragula/components/ember-dragula-container';
+import SwipableListMixin from 'action-chat-client/mixins/swipable-list';
+
+export default emberDragulaContainer.extend(SwipableListMixin, {
+});
```

Comment:
The 'class' should follow the convention of using capitalised camel case ie. "EmberDragulaContainer"